util/dvsim/FlowCfg.py

```diff
@@ -392,6 +394,19 @@ def print_list(self):
         for item in self.cfgs:
             item._print_list()
 
+    # function to prune only selected cfgs to build and run
+    # it will return if the object is not a master_cfg or -select_cfgs is empty
+    def prune_selected_cfgs(self):
+        if self.is_master_cfg == 0 or self.select_cfgs == []:
+            return
+        else:
+            remove_cfgs = []
+            for item in self.cfgs:
+                if item.dut not in self.select_cfgs:
```

Comment:
I think this should be `item.name` instead.

Reply:
I see, thanks Sri. I updated to `name`.